### PR TITLE
[windows] Add filters to visuals for only AppLocker data to be displayed

### LINKED ISSUE #8969: Dashboard issues with "[Windows AppLocker] Audited and Blocked Applications" dashboard as provided by "Windows" integration 1.44.1
there appears to be an insufficient filter on the "Top File names by Application Execution Count" (especially since I only have a single host that is reporting AppLocker events right now)
image
When I examine the data in Discover, I see that it is looking in logs-* for "File.name":*
image

I suggest adding the filter for "event.provider:Microsoft-Windows-Applocker" (which I had to edit in Lens)
image

A similar filter probably needs to be added to file publishers based on FQBN by application execution count, and honestly, probably all of the panels

### PR DESCRIPTION
**Pull request by nicpenning**
- Bug

This fixes the visuals in the dashboard for AppLocker so only relevant data is displayed. See: https://github.com/elastic/integrations/issues/8969

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

- Closes #8969

### CODE REVIEW

**Comment by nicpenning** [comment]
This is ready to test.

**Review by marc-gr — DISMISSED** [review]

**Comment by marc-gr** [comment]
I think `elastic-package check` is required to run and commit the changes it does.

**Comment by nicpenning** [comment]
Okay - So I need to run that command and then push the changes up?

**Comment by nicpenning** [comment]
elastic-package check has been applied and committed. Please test again!

**Comment by nicpenning** [comment]
I cannot see the build issues now that it's been moved to buildkite. Any chance this can be opened up to community members contributing?

**Comment by nicpenning** [comment]
Any updates on why this failed? I am kinda blind over here.

**Comment by strawgate** [comment]
I see a failure of:

```
Error: can't install the package: could not zip-install package; API status code = 500; response body = {"statusCode":500,"error":"Internal Server Error","message":"Migration function for version 7.11.0 threw an error"}
```

I'll see if we can get someone to comment about plans for access

**Comment by nicpenning** [comment]
Thank you. Any idea how that issue can be resolved?

**Comment by marc-gr** [comment]
> Is this issue sourced from using 8.12.0 when I checked the package?
> 
> @strawgate - is it still the same error?

Could be, I checked and for 8.12 it passes alright, but CI uses the minimum supported version which is 8.8.0. If you used 8.12 to do the changes that might be the issue. I'd suggest doing them with an 8.8.0 stack (`elastic-package stack up -v -d --version=8.8.0`)

**Comment by nicpenning** [comment]
Bummer - Okay, I can do that. Perhaps it is a deeper issue, but it is concerning we must use such an older version of the stack. In an example of a visualization that only exists in the latest, this would likely then be a blocker and not something we can add until the elastic-package tool can keep up? I can reference this issue there if that will help with this side issue. Please let me know!

File: packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json [hunk 1]
```diff
@@ -44,4 +44,4 @@
             }
         }
     ]
-}
+}
```
Comment (rdner) [inline packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json:47]: All whitespace changes should be reversed, so they don't pollute file history.

File: packages/windows/manifest.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: windows
 title: Windows
-version: 1.44.1
+version: 1.44.2
```
Comment (rdner) [inline packages/windows/manifest.yml:3]: This version change will cause a conflict once https://github.com/elastic/integrations/pull/8976 is merged

**Review by rdner — CHANGES_REQUESTED** [review]

**Comment by marc-gr** [comment]
I'll take care of fixing this one since not having access to CI complicates things for @nicpenning . Thanks for the work done @nicpenning will update this PR with the fix. 👍

File: packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json (hunk 1 above)
Comment (nicpenning) [reply packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json:47]: Thanks, @rdner for the awareness. How should I handle incorrect white space in the future? I believe the elastic-package tool corrected this. 

@efd6 what's your take on this? It seems like I can never win on the whitespace things haha!! 😅

Comment (rdner) [reply packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json:47]: If it's caused by automation it's okay to leave it. Looks like this code was somehow merged before the tool started taking care of code formatting.

File: packages/windows/manifest.yml (hunk 2 above)
Comment (rdner) [reply packages/windows/manifest.yml:3]: Now merged, we need to bump the version in this PR to 1.44.3.

File: packages/windows/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.44.3"
```
Comment (rdner) [inline packages/windows/changelog.yml:2]: Needs to be set in https://github.com/elastic/integrations/blob/655207ba9c8eff24eb806a06864f181c390d3367/packages/windows/manifest.yml#L3 too

File: packages/windows/manifest.yml (hunk 2 above)
Comment (rdner) [reply packages/windows/manifest.yml]: Done by resolving conflicts in https://github.com/elastic/integrations/pull/8993/commits/7b5b2e847593ece9e985925cabb13b474356930c

File: packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json (hunk 1 above)
Comment (nicpenning) [reply packages/windows/data_stream/forwarded/_dev/test/pipeline/test-security-4662.json:47]: You caught me 😀 

That's exactly what happened. I will keep that in mind. Thank you!

**Comment by nicpenning** [comment]
👋 
Any updates here?

**Review by rdner — APPROVED** [review]

**Review by marc-gr — APPROVED** [review]

**Comment by nicpenning** [comment]
🎉

**Pull request merged**